packages/gnat/.vscode/launch.json

```diff
@@ -38,6 +38,25 @@
       ],
       "request": "launch",
       "console": "integratedTerminal"
-    }    
+    },
+    {
+      "name": "Confinement",
+      "type": "python",
+      "request": "launch",
+      "module": "confinement.confinement",
+      "console": "integratedTerminal",
+      // Add NATIONAL_PROJECT, INPUTS_FOLDER, DOWNLOAD_FOLDER to your .env file
+      "args": [
+        "${input:HUC}",
+        "D:\\NAR\\Projects\\Confinement\\flowlines.shp",
```

Comment:
Totally minor thing but I wouldn't leave a hardcoded path in here. Use the .env file (take a look at rs_context for examples)